### PR TITLE
FIX: focalpoint

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
This PR fixes https://github.com/mne-tools/mne-python/issues/9007 by changing the camera's default focal point.

### CODE REVIEW

File: mne/viz/backends/_pyvista.py
Comment (larsoner) [inline mne/viz/backends/_pyvista.py:980]:
```
>>> np.array([0, 0, 0]) == 'auto'
<stdin>:1: FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
False
```
Typically we handle this with:
```
_validate_type(focalpoint, (str, None, np.ndarray), 'focalpoint')
if isinstance(focalpoint, str):
    _check_option('focalpoint', focalpoint, ('auto',), extra='when a string')
    ...
elif focalpoint is None:
    ...
else:
    ...
```

**Comment by larsoner** [comment]
Weird rendering problems:

https://25630-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/source-modeling/plot_background_freesurfer_mne.html#using-freesurfer-s-surface-reconstructions

**Comment by GuillaumeFavelier** [comment]
Those are really scary. I think it got reported but I didn't take care of it yet. I agree it's probably resize event or something.

My next focus will be between this and multiplotter.

**Comment by larsoner** [comment]
@GuillaumeFavelier can you push a commit with `[circle full]` and then we can look to see if things look okay?

**Comment by GuillaumeFavelier** [comment]
BTW, the results are available on:

https://26478-1301584-gh.circle-artifacts.com/0/dev/index.html

**Comment by GuillaumeFavelier** [comment]
I still see issues:

https://26478-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/source-modeling/plot_eeg_mri_coords.html

image

**Comment by GuillaumeFavelier** [comment]
Circle probably uses PyVista master since it breaks already. I'll open a PR to get ready for those changes.

**Comment by larsoner** [comment]
@GuillaumeFavelier any chance to return to this? It's going to be a blocker for 0.23

**Comment by GuillaumeFavelier** [comment]
TBH I was counting on https://github.com/mne-tools/mne-python/pull/8997 to fix it indirectly but considering the slow progress, it might not be enough for the next release. I'll give it max priority then :+1:

**Comment by larsoner** [comment]
I think #8997 is a good one to merge right after release rather than right before it. It's a big change and we want some months of testing and use ourselves to make sure it's stable

**Comment by GuillaumeFavelier** [comment]
I think it's worth a `[circle full]` after https://github.com/mne-tools/mne-python/pull/9318 is merged.

**Comment by GuillaumeFavelier** [comment]
`tutorials/forward/10_background_freesurfer.py`: https://27738-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/forward/10_background_freesurfer.html?highlight=freesurfer

main | PR
--|--
image | image

`tutorials/forward/50_background_freesurfer_mne.py`: https://27738-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/forward/50_background_freesurfer_mne.html?highlight=freesurfer

main | PR
--|--
image | image

For reference:

`examples/connectivity/sensor_connectivity.py`: https://27738-1301584-gh.circle-artifacts.com/0/dev/auto_examples/connectivity/sensor_connectivity.html?highlight=sensor%20connectivity
image

And locally for `tutorials/inverse/60_visualize_stc.py`:

image
image
image

**Comment by larsoner** [comment]
LGTM -- ready to go, then?

**Comment by GuillaumeFavelier** [comment]
Ready on my end

**Review by larsoner — APPROVED** [review]
LGTM will merge once CIs come back happy!

**Comment by larsoner** [comment]
Thanks @GuillaumeFavelier !

**Pull request merged**
